src/components/App/index.test.tsx

```diff
@@ -64,3 +65,37 @@ it('shows dialogs in the queue', async () => {
 
   expect(getByText('wowah')).not.toBeNull();
 });
+
+it('redirects from url /functions to correct logs url', async () => {
+  const store = configureStore();
+  const history = createMemoryHistory({
+    initialEntries: ['/functions'],
+  });
+  render(
+    <Provider store={store}>
+      <Router history={history}>
+        <App />
+      </Router>
+    </Provider>
+  );
+  await act(() => delay(100)); // Wait for tab indicator async DOM updates.
+  const { pathname, search } = history.location;
+  const redirectedCompleteUrl = pathname + search;
```

Comment:
nit: eliminate the variable `redirectedCompleteUrl` and just put `pathname + search` into the expect statement